source/blog/2016-11-28-ember-2-10-released.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Ember 2.10 and 2.11.0-beta.1 Release
+author:
+tags: Releases
+---
+
+## Ember.js
+
+
+
+
+## Ember Data
+
+Ember Data 2.10.0, a minor version release of Ember Data, is
+released. This release represents the work of over 11 direct
+contributors, and over 27 commits.
+
+Ember Data 2.11-beta.1, the first beta release of Ember Data 2.11.0,
+is also being released. It is expected it be published as a final
+release in roughly 6 weeks.
+
+### Changes in Ember Data 2.10
+
+There have been no user facing changes in Ember Data 2.10. Under the
+hood, [@runspired](https://github.com/runspired) has instrumented
+Ember Data with some internal tools to make it easier to track the
+performance of Ember Data. This instrumentation is stripped from the
+production build of Ember Data and should have no impact on existing
+apps that rely on Ember Data.
+
+[@runspired](https://github.com/runspired) also
+[improved the messaging](https://github.com/emberjs/data/pull/4578)
+around assertions and warnings when an adapter responses with no `id`
+for a record.
+
+### Upcoming changes in Ember Data 2.11.0-beta.1
+
+Ember Data 2.11 is already shaping up to be a big release with many
+performance improvements and documentation updates.
+
+#### HasMany Computed Property Keys
+
+One of the performance improvements included in the Ember Data
+2.11.0-beta.1 release fixed an issue where Ember Data was doing too
+much work when records in a `DS.hasMany()` changed. As a result of
+this extra work Ember Data would incorrectly invalidate computed
+properties that depended on these `hasMany` relationships. After this
+fix you may need to up computed property keys that depended on
+`record.hasManyRelationshipName` to
+`record.hasManyRelationshipName.length` or
+`record.hasManyRelationshipName.[]`.
+
+#### Deprecate store.serialize
+
+Ember Data now issues depreaction warnings for `store#serialize()` and
+`adapter#serialize()`. Instead it is recommended that you use
+`record.serialize()` in place of `store.serialize(record)` or
+`snapshot.serialize()` in place of
+`adapter.serialize(snapshot)`. `store#serialize()` and
+`adapter#serialize()` will be removed in Ember Data 3.0.0.
```

Comment:
https://github.com/emberjs/data/blob/master/addon/-private/system/store.js#L2595-L2609

^ newly deprecated methods `lookupAdapter` and `lookupSerializer`, these were never fully public or private and should get a mention.